gymnasium/envs/mujoco/mujoco_env.py

```diff
@@ -360,7 +360,7 @@ def __init__(
         height: int = DEFAULT_SIZE,
         camera_id: Optional[int] = None,
         camera_name: Optional[str] = None,
-        default_camera_config: Optional[Dict[str, Union[float, int]]] = None,
+        default_camera_config: Optional[Dict[str, Union[float, int]]] = {},
```

Comment:
Is there a reason why this was changed from `None`. 
It is bad practice to have mutable types as arguments

Reply:
@pseudo-rnd-thoughts  reverted

Reply:
Thanks for changing